src/System.Collections.NonGeneric/tests/CollectionBaseTests.cs

```diff
@@ -241,16 +238,16 @@ public static void TestCopyTo()
         }
 
         [Fact]
-        public static void TestCopyTo_Invalid()
+        public static void CopyTo_Invalid()
         {
             MyCollection collBase = CreateCollection(100);
             var fooArr = new Foo[100];
-            Assert.Throws<ArgumentOutOfRangeException>(() => collBase.CopyTo(fooArr, -1)); // Index < 0
-            Assert.Throws<ArgumentException>(() => collBase.CopyTo(fooArr, 50)); // Index + fooArray.Length > collBase.Count
+            Assert.Throws<ArgumentOutOfRangeException>("dstIndex", () => collBase.CopyTo(fooArr, -1)); // Index < 0
+            Assert.Throws<ArgumentException>("", () => collBase.CopyTo(fooArr, 50)); // Index + fooArray.Length > collBase.Count
```

Comment:
I'm not sure if there's a preference between string.Empty and "" tests, but sure!